tests/rptest/scale_tests/partition_balancer_scale_test.py

```diff
@@ -161,3 +163,128 @@ def all_reconfigurations_done():
+            consumers = 8
+            partitions_count = 200
+            max_concurrent_moves = 200
+            timeout = 300
+
+        # set max number of concurrent moves
+        self.redpanda.set_cluster_config(
+            {"partition_autobalancing_concurrent_moves": max_concurrent_moves})
+
+        topic = TopicSpec(partition_count=partitions_count,
+                          replication_factor=replication_factor)
+        self.client().create_topic(topic)
+
+        self._start_producer(topic.name, message_cnt, message_size)
+        self._start_consumer(topic.name, message_size, consumers=consumers)
+        mb = 1024 * 1024
+        self.logger.info(
+            f"waiting for {(message_size*message_cnt) / mb} MB to be produced to "
+            f"{partitions_count} partitions ({((message_size*message_cnt) / mb) / partitions_count} MB per partition"
+        )
+        # wait for the partitions to be filled with data
+        self.producer.wait_for_acks(message_cnt // 2,
+                                    timeout_sec=timeout,
+                                    backoff_sec=5)
+
+        admin = Admin(self.redpanda)
+        brokers = admin.get_brokers()
+        # decommission one of the nodes
+        decommissioned_id = random.choice(brokers)['node_id']
+        self.logger.info(
+            f"cluster brokers: {brokers}, decommissioning: {decommissioned_id}"
+        )
+        admin.decommission_broker(decommissioned_id)
+
+        def decommission_ended():
+            replicas = self.node_replicas([topic.name], decommissioned_id)
+            self.logger.debug(
+                f"decommissioned node {decommissioned_id} hosts {len(replicas)} replicas"
+            )
+            if len(replicas) != 0:
+                return False
+            all_brokers = set()
+            for n in self.redpanda.nodes:
+                current_id = self.redpanda.node_id(n)
+                if current_id == decommissioned_id:
+                    continue
+                brokers = admin.get_brokers(node=n)
+                for i in [b['node_id'] for b in brokers]:
+                    all_brokers.add(i)
+            return decommissioned_id not in all_brokers
+
+        wait_until(decommission_ended, timeout, 5)
+        # restart node
+        to_restart = None
+        for n in self.redpanda.nodes:
+            current_id = self.redpanda.node_id(n)
+            if current_id == decommissioned_id:
+                to_restart = n
+                break
+
+        def seed_servers_for(node):
+            seeds = map(
+                lambda n: {
+                    "address": n.account.hostname,
+                    "port": 33145
+                }, self.redpanda.nodes)
+
+            return list(
+                filter(lambda n: n['address'] != node.account.hostname, seeds))
+
+        self.redpanda.stop_node(to_restart)
+        self.redpanda.clean_node(to_restart)
+        self.redpanda.start_node(to_restart,
```

Comment:
In case you're forming the seeds to avoid a root node wipe, you can use

```
self.redpanda.start_node(to_restart, omit_seeds_on_idx_one=False)
```